### ISSUE TITLE
Vite dev server won't automatically pick up new Sass files imported via Glob

### ISSUE THREAD

**Issue by cmalven** [cmalven/vite-plugin-sass-glob-import#4]
Sass files that are imported via globbing aren't automatically picked up by the Vite dev server, so if you add a new Sass file that is only imported via globbing, you won't see the styles from that file apply until you either restart the Vite dev server OR make a change to the Sass file that does the glob importing.

I'm not sure it makes sense for a solution to this to be baked into this plugin - I think a more reasonable long-term solution is that [vite-plugin-restart](https://www.npmjs.com/package/vite-plugin-restart) adds support for restarting the server when new files are created.

**Comment by raray** [comment]
They added support for restarting on `create` and `unlink` in `vite-plugin-restart`, but using that to watch Sass files will cause a server restart on every change instead of an HMR.

**Comment by cmalven** [comment]
@raray Yep, that's unfortunate. I was hoping they'd expand the API so that you could listen for `create` and `unlink` events on specific grep patterns. Seems like that's unlikely to happen, so I might need to consider baking something into this plugin. Not a lot of free time to make that happen though, so I'd welcome a PR from anyone interested.
